src/test/java/com/exasol/adapter/document/UdfCallBuilderTest.java

```diff
@@ -33,7 +26,7 @@ void testBuildForEmptyPlan() throws IOException {
         final RemoteTableQuery remoteTableQuery = getRemoteTableQueryWithOneColumns();
         final QueryPlan queryPlan = new EmptyQueryPlan();
         final String udfCallSql = UDF_CALL_BUILDER.getUdfCallSql(queryPlan, remoteTableQuery);
-        assertThat(udfCallSql, equalTo("SELECT * FROM (VALUES (CAST(NULL AS VARCHAR(123)))) WHERE FALSE"));
+        assertThat(udfCallSql, equalTo("SELECT * FROM (VALUES (CAST(NULL AS  VARCHAR(123)))) WHERE FALSE"));
```

Comment:
Is this needed?

Reply:
Well, actually not. But sql-statement-builder by accident adds two spaces and I did not want to investigate.